src/app/context/load.ts

```diff
@@ -32,6 +33,20 @@ export async function loadFiles(
         : await fileCollectionFromFiles(files);
     const measurements = await loadMeasurements(fileCollection, options);
     dispatch({ type: 'ADD_MEASUREMENTS', payload: measurements });
+
+    let kind: MeasurementKind | null = null;
+    for (const k in measurements) {
+      if (measurements[k].entries.length > 0) {
+        kind = k as MeasurementKind;
+        break;
+      }
+    }
+    if (kind) {
+      dispatch({
+        type: 'SELECT_MEASUREMENT',
+        payload: { id: measurements[kind].entries[0].id, kind },
+      });
+    }
```

Comment:
I think this should be done inside the ADD_MEASUREMENTS action, and only if there is nothing selected yet. @stropitek ?

Reply:
Yes I think this logic can be moved to the ADD_MEASUREMENTS action

Reply:
Fixed, thanks.